lib/client.js

```diff
@@ -355,8 +356,8 @@ Client.prototype.createTopics = function (topics, isAsync, cb) {
     cb = isAsync;
     isAsync = true;
   }
+  validateKafkaTopics(topics);
```

Comment:
Just realized something. if this fails validation and `isAsync` is true then user will never be notified of the error through the callback. I think we should change this to account for that case.